### DISCUSSION TITLE
Add a search to licensed repositories

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14471] [Other Feature Feedback, Questions, & Ideas]
Hi Github!

I would like this feature, because if you have a lot of repositories like me, a license search is extremely useful for finding specific repositories.

#### 1. ui/ux
![image](https://user-images.githubusercontent.com/95204402/162201145-91c67d36-65ce-44ab-b591-42fdf06f98b3.png)

#### 2. user stories
1. As a user I would like to filter the repositories by license: MIT, GPL... among other licenses that are registered in each repository
2. As a user I would like to click on a button where after I click, filter the repositories by license

#### 3. notes
1. Hope to contribute positively to Github
2. I didn't find anything talking about this feature... I didn't find any requests for this feature

**Comment by xobhuv-0cAksy-cygqov** [comment]
> Hi Github!
> 
> I would like this feature, because if you have a lot of repositories like me, a license search is extremely useful for finding specific repositories.
> 
> #### 1. ui/ux
> ![image](https://user-images.githubusercontent.com/95204402/162201145-91c67d36-65ce-44ab-b591-42fdf06f98b3.png)
> 
> #### 2. user stories
> 1. As a user I would like to filter the repositories by license: MIT, GPL... among other licenses that are registered in each repository
> 2. As a user I would like to click on a button where after I click, filter the repositories by license
> 
> #### 3. notes
> 1. Hope to contribute positively to Github
> 2. I didn't find anything talking about this feature... I didn't find any requests for this feature
